src/components/toolbox/passphraseInput/passphraseInput.css

```
@@ -1,24 +1,84 @@
 @import '../../../app/variables.css';
 @import '../../../app/mixins.css';
 
+.wrapper {
+  margin: 0;
+  position: relative;
+}
+
 .inputs {
+  width: 100%;
+  display: flex;
   flex-wrap: wrap;
+  flex-direction: row;
+  border: 1px solid var(--dark-border-color);
+  border-radius: 3px;
+  background-color: var(--color-white);
+  padding: 0px 24px;
+  box-sizing: border-box;
+  outline: none;
+
+  &:focus-within {
+    border-color: var(--color-ultramarine-blue);
+  }
+
+  & > .inputContainer {
+    width: calc(100% / 6);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    flex-grow: 1;
+    box-sizing: border-box;
+    color: var(--color-ghost);
+    margin: 15px 0 20px;
```

I think the margin in the design is:
```suggestion
    margin: 16px 0;
```